### DISCUSSION TITLE
Monitoramento da Api

### DISCUSSION THREAD

**Discussion by trainneEg7** [HashLoad/horse#312] [General]
Olá, preciso desenvolver uma aplicação para monitorar a api, sendo quantos requisições aconteceu em determinado endpoint, quantos teve no dia, qual o consumo de memoria. monitoramento em relação a isso. 

por acaso tem algum middleware para isso? é possivel desenvolver em delphi?

**Comment by viniciussanchez** [comment]
Como respondido no privado, existe algo que é o middleware de healthcheck: https://github.com/CarlosHe/horse-healthcheck
